spec/controllers/api/v1/book_suggestion_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+require 'support/shared_context'
+
+describe Api::V1::BookSuggestionController, type: :controller do
+  include_context 'Authenticated User'
+  include Devise::Test::ControllerHelpers
+  describe 'POST #create' do
+    subject(:post_create) { post :create, params: book_sugg }
```

Comment:
👏

Reply:
I'll try to add `type: :controller` to see if it works

Reply:
It worked!